src/layer/TectonicPlatesLayer.js

```diff
@@ -50,7 +50,7 @@ define([
 
     TectonicPlatesLayer.prototype = Object.create(RenderableLayer.prototype);
 
-    Object.defineProperties(SurfacePolygon.prototype, {
```

Comment:
I would fix that in a separate pull request unless this is impacting WKT.

Reply:
I agree with @AkeluX. This is an important change, but it's much better to keep change sets independent.